roles/smallfile/templates/smallfilejob.other_parameters.yaml.j2

```diff
@@ -1,6 +1,14 @@
 
 
 top: {{ smallfile_path }}/smallfile_test_data
+ 
+{% if workload_args.auto_pause is defined %}
```

Comment:
Please add new args to `docs/smallfile.md`

Reply:
I resolved this differently, I linked to github smallfile repo.  Do not want to have to maintain in 2 or more places.  Also updated other parts of this doc.